src/main/java/core/basesyntax/service/Parser.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface Parser {
+    List<String[]> parseData(List<String> rawData);
```

Comment:
```suggestion
    List<FruitTransaction> parseData(List<String> rawData);
```
String[] - is also raw Data - it is not java object. We need to have List<FruitTransaction>